src/extensions/renderer/canvas/drawing-edges.js

```diff
@@ -253,7 +250,7 @@ CRp.drawArrowShape = function( edge, arrowType, context, fill, edgeWidth, shape,
   if( context.beginPath ){ context.beginPath(); }
 
   if( !pathCacheHit ){
-    shapeImpl.draw( context, size, angle, translation );
+    shapeImpl.draw( context, size, angle, translation, edgeWidth );
```

Comment:
This is where I send the edgeWidth to the 'triangle-cross' draw function.  Again, thoughts on this?